src/app/Recordings/ActiveRecordingsTable.tsx

```diff
@@ -927,6 +928,14 @@ export const ActiveRecordingRow: React.FC<ActiveRecordingRowProps> = ({
     handleLoadAnalysis();
   }, [isExpanded, handleLoadAnalysis]);
 
+  React.useEffect(() => {
+    addSubscription(
+      context.notificationChannel.messages(NotificationCategory.ReportSuccess).subscribe(() => {
+        handleLoadAnalysis();
```

Comment:
If we have that "report job UUIDs" list somewhere, like as an accessible member of the Report service, then we can filter these notifications to only ones that include an expected UUID and only react to those ones. That way we don't trigger the `handleLoadAnalysis()` function if we observe a `ReportSuccess` notification that has a wrong UUID, ie. which was emitted for some other reason (like another client requested a report on another recording).